website/src/content/concepts/oauth-scopes-and-user-permissions.mdx

```diff
@@ -30,6 +30,37 @@ Notice here how the OAuth Scopes are grouped by the two fields `view` and `manag
 
 This grouping determines the **mapping and relation between OAuth Scopes and user permissions**.
 
+# Additional OAuth Scopes
```

Comment:
Wondering if we should maybe use the wording "permission group" here and have it as a subsection of `OAuth Scopes`.

Reply:
Also, we should include the info about the min version to use the granular permission groups.

```
<Info>

This feature is available from version `21.21.0` onwards.

</Info>
```